reflex/ivars/sequence.py

```diff
@@ -1029,42 +1486,67 @@ def array_contains_operation(haystack: ArrayVar, needle: Any | Var):
     frozen=True,
     **{"slots": True} if sys.version_info >= (3, 10) else {},
 )
-class ToStringOperation(CachedVarOperation, StringVar):
+class ToStringOperation(StringVar):
```

Comment:
seems like we could use a new mixin for `ToXOperation` that removes the `__getattr__`, `__post_init__`, and `__hash__` boilerplate